CMakeLists.txt

```diff
@@ -345,60 +345,66 @@ set(STRIP_TABLE_CMD "awk '/Problem Summary/ { while (!match($0, /termination/)){
 # If running in a BSUB allocaation, use jrun commands and make sure we are
 # requesting the correct resources.
 if(HIOP_TEST_WITH_BSUB)
-  set(RUNCMD "jsrun -n 1")
+  set(RUNCMD "jsrun" "-n" "1")
   set(MPICMD "jsrun")
   if(HIOP_USE_GPU)
-    set(MPICMD "${MPICMD} -g 1")
-    set(RUNCMD "${RUNCMD} -g 1")
+    set(MPICMD ${MPICMD} "-g" "1")
+    set(RUNCMD ${RUNCMD} "-g" "1")
   endif()
 else()
-  set(RUNCMD "") # No special command is needed to run this program
   set(MPICMD "mpirun")
+  if(HIOP_USE_MPI)
+    set(RUNCMD "mpirun" "-n" "1")
+    set(RUNCMD ${RUNCMD} "-mca" "oob" "tcp")
+    set(MPICMD ${MPICMD} "-mca" "oob" "tcp")
```

Comment:
Are these specific to newell? Can these cause problems elsewhere? I don't think the line
```
set(RUNCMD ${RUNCMD} "-mca" "oob" "tcp")
```
makes much sense either.

Perhaps we should allow the user to pass an argument or set an environment variable to add arguments to runcmd/mpicmd.

Reply:
These are probably only specific to PNNL clusters, and so I agree that we should probably remove these in favour of the optional input you suggested. Since the options are just suppressing warnings, would it suffice to just remove the options? Is there a particular use case that would require optional options being passed to mpi/jsrun?

Reply:
I'd add a cmake line like:
```cmake
set(HIOP_EXTRA_MPI_FLAGS "" CACHE STRING "Extra arguments to mpiexec when running tests")
```
and then in the build script under the newell-specific section, we can add
```shell
extra_cmake_args="$extra_cmake_args -DHIOP_EXTRA_MPI_FLAGS=\"-mca oob tcp\""
```

Reply:
I have implemented this suggestion in my most recent commit. The only thing that I had to change was passing the `extra_cmake_args` in a slightly different way:
```shell
extra_cmake_args="$extra_cmake_args -DHIOP_EXTRA_MPI_FLAGS=-mca;oob;tcp;"
```